executor/prepared_test.go

```diff
@@ -104,6 +104,25 @@ func (s *testSuite1) TestPrepareStmtAfterIsolationReadChange(c *C) {
 	c.Assert(tk.Se.GetSessionVars().PreparedStmts[1].(*plannercore.CachedPrepareStmt).NormalizedPlan, Equals, "")
 }
 
+func (s *testSuite9) TestPreparedStmtWithHint(c *C) {
+	// see https://github.com/pingcap/tidb/issues/18535
+	store, dom, err := newStoreWithBootstrap()
```

Comment:
No need to new store.

Reply:
I originally deleted it, however it turns out that I need to use a new `domain`, I created it again.